cpu/sam0_common/periph/rtc.c

```diff
@@ -41,13 +43,50 @@ static rtc_state_t rtc_callback;
 * Thanks to this, the user will be able to set time in 2000's*/
 static uint16_t reference_year = 100;
 
-void rtc_init(void)
+static void wait_syncbusy(void)
```

Comment:
static function should be named with _ prefix
`_wait_syncbusy()`
This is the same for other static functions within this page.

Reply:
See our [coding convention](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions) if something is unclear to you